content/en/case-studies/adidas/index.html

```diff
@@ -68,13 +68,13 @@ <h2>In recent years, the adidas team was happy with its software choices from a
 </section>
 ​
 ​
-  <div class="article image overlay" style="background-image: url('/images/Adidas3.png');padding-top:2% !important">
+  <div class="article image overlay" style="background-image: url('/images/Adidas3.png');">
   <div class="quotetext">
```

Comment:
@jkld11 , is this url to .png file correct? Should there be a corresponding .svg file?

Reply:
@kbhawkey Hi, yes, those urls are correct. They are the images that are the background for the banners.

Reply:
The error message:
8:28:59 PM: ERROR 2019/11/12 01:28:59 Case Studies: Missing logo for adidas. Put a PNG with the word 'logo' in the filename into "case-studies/adidas/"